app/src/main/res/values/styles.xml

```diff
@@ -99,7 +99,10 @@
     </style>
 
     <style name="NavigationButton" parent="Widget.AppCompat.ImageButton">
+        <item name="android:layout_width">@dimen/button_size</item>
+        <item name="android:layout_height">@dimen/button_size</item>
+        <item name="android:layout_marginEnd">@dimen/button_right_margin</item>
```

Comment:
Now that width/height/marginEnd are only used in one place, I don't think we need them to read from dimens anymore.